pyuvdata/utils.py

```diff
@@ -2049,7 +2049,7 @@ def calc_uvw(
         # GHA -> Hour Angle as measured at Greenwich (because antenna coords are
         # centered such that x-plane intersects the meridian at longitude 0).
         if to_enu:
-            # Unphased coordinates appear to be stored in ENU coordinates -- that's
+            # Unprojected coordinates appear to be stored in ENU coordinates -- that's
```

Comment:
"coordinates appear to be stored in ENU coordinates" -> "coordinates are given in the ENU convention"